jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -962,6 +962,62 @@ ecma_op_to_length (ecma_value_t value, /**< ecma value*/
 #endif /* ENABLED (JERRY_ES2015) */
 } /* ecma_op_to_length */
 
+#if ENABLED (JERRY_ES2015)
+/**
+ * CreateListFromArrayLike operation.
+ * Different types are not handled yet.
+ *
+ * See also:
+ *          ECMA-262 v6, 7.3.17
+ *
+ * @return ecma_collection_t if successful
+ *         NULL otherwise
+ */
+ecma_collection_t *
+ecma_op_create_list_from_array_like (ecma_value_t arr) /**< array value */
+{
+  /* 1. */
+  if (ECMA_IS_VALUE_ERROR (arr))
```

Comment:
I think this step can be discarded, since it's not usual to pass error value to methods. (I'd rather assert that the argument is not an error.)